src/Symfony/Component/HttpKernel/Exception/ControllerDoesNotReturnResponseException.php

```diff
@@ -67,9 +67,15 @@ private function parseControllerDefinition(callable $controller): ?array
         if (\is_object($controller)) {
             $r = new \ReflectionClass($controller);
 
+            try {
+                $line = $r->getMethod('__invoke');
```

Comment:
:thinking:

Reply:
Yeah, missed `->getEndLine()`. I will change it asap.

Reply:
Done